tests/integ/scala/test_function_suite.py

```diff
@@ -2229,6 +2229,7 @@ def test_as_object(session):
 
 
 def test_timestamp_tz_from_parts(session):
+    session.sql("alter session set timezone = 'America/Los_Angeles'").collect()
```

Comment:
Is it possible to set it with statement params? https://github.com/snowflakedb/snowpark-python/blob/7c6336bc2ee1761e1fad23da2432e7a131becaa9/src/snowflake/snowpark/dataframe.py#L535

If you change the parameter here, the following tests will also be impacted.